swig/include/gdalconst.i

```diff
@@ -205,11 +205,19 @@
 %constant char *DCAP_RASTER     = GDAL_DCAP_RASTER;
 %constant char *DCAP_VECTOR     = GDAL_DCAP_VECTOR;
 %constant char *DCAP_GNM     = GDAL_DCAP_GNM;
+%constant char *DCAP_CREATE_LAYER = GDAL_DCAP_CREATE_LAYER;
```

Comment:
those constants should also be ported in the #else clause (after line 233) for the Java and C# bindings

Reply:
done